### PR TITLE
Add Logical Deployment Diagram for OSMT

### PR DESCRIPTION
**Pull request by JohnKallies**

### CODE REVIEW

File: docs/osmt_logical_deployment_diagram.mdj [hunk 1]
```diff
@@ -0,0 +1,7596 @@
...
+											"_parent": {
+												"$ref": "AAAAAAGEV910z/Tqajg="
+											},
+											"fillColor": "#d8f2ff",
+											"font": "Arial;13;1",
+											"left": 81,
+											"top": 377,
+											"width": 144.4853515625,
+											"height": 13,
+											"text": "Brower - OSMT User"
```
Comment (wgu-edwin) [inline docs/osmt_logical_deployment_diagram.mdj:2495]: Is "Brower" a typo for browser?

**Review by wgu-edwin — COMMENTED** [review]
Only found that one typo. Everything else looks good to me

File: docs/osmt_logical_deployment_diagram.mdj (hunk 1 above)
Comment (JohnKallies) [reply docs/osmt_logical_deployment_diagram.mdj:2495]: Why.... yes it is.

**Review by jchavez137 — APPROVED** [review]
looks good to me

**Pull request merged**